core/arch/arch.c

```diff
@@ -728,8 +726,8 @@ d_r_arch_init(void)
     /* We're allocating a reachable heap variable in order to be able to use a more
      * compact rip-rel load in SIMD restore/save gencode.
      */
-    d_r_avx512_code_in_use =
-        heap_reachable_alloc(GLOBAL_DCONTEXT, 1 HEAPACCT(ACCT_OTHER));
+    d_r_avx512_code_in_use = heap_reachable_alloc(
+        GLOBAL_DCONTEXT, sizeof(d_r_avx512_code_in_use) HEAPACCT(ACCT_OTHER));
```

Comment:
Wrong size: need * (as was in review comment)

Reply:
Fixed.